src/interfaces/assistants_web/src/components/Shared/HotKeys/HotKeysDialog.tsx

```diff
@@ -61,10 +89,9 @@ export const HotKeysDialog: React.FC<Props> = ({ isOpen, close, options = [] })
           leaveFrom="opacity-100"
           leaveTo="opacity-0"
         >
-          <div className="transition-o`pacity fixed inset-0 bg-volcanic-300/20 backdrop-blur-sm" />
+          <div className="fixed inset-0 bg-[#B3B3B3]/60 transition-opacity dark:bg-[#1C1C1C]/80" />
```

Comment:
Can we use `volcanic-700` for light and ask for a theme color for dark?

Reply:
yep will do in follow up! merging now to unblock deploy